doc/whats_new.rst

```diff
@@ -63,6 +63,17 @@ Decomposition, manifold learning and clustering
   division on Python 2 versions. :issue:`9492` by
   :user:`James Bourbeau <jrbourbeau>`.
 
+Neighbors
+
+- Fix bug so that the method ``predict`` for 
+  :class:`neighbors.RadiusNeighborsRegressor` can handle empty neighbor set 
+  when using non uniform weights. Also raises a new warning, irrespective of 
+  weighting, when no neighbors are found for one or more samples. Moreover, 
```

Comment:
maybe just "... samples, in place of unclear numpy errors."

Reply:
Good suggestion, thanks